README.md

```diff
@@ -156,10 +156,17 @@ HOF-bootstrap accepts the following options so a developer can customise element
 
 ### Options
 
-- `views`: Location of the base views relative to the root of your project. Defaults to 'views'. Set `views` to `false` if not present.
+- `views`: Location of the base views relative to the root of your project.
+  - Will not error if it can't find any views in the root of your project.
+  - Looks for views - optionally set in your [route options](#route-options).
+  - Always loads views from [hof-template-partials.views](https://github.com/UKHomeOfficeForms/hof-template-partials/tree/master/views).
```

Comment:
[hof-template-partials.views] are always loaded.
So are base views and route views, if they a path to them if specified in the options, and the path is valid